ui/src/organizations/components/AddMembersForm.tsx

```diff
@@ -1,31 +1,55 @@
 // Libraries
-import React, {PureComponent} from 'react'
+import React, {PureComponent, ChangeEvent} from 'react'
 
 // Components
-import {Button, ButtonType} from '@influxdata/clockface'
-import {Form, Grid} from 'src/clockface'
+import {Button, ButtonType, IconFont} from '@influxdata/clockface'
+import {Form, Grid, Input, Columns} from 'src/clockface'
 import SelectUsers from 'src/organizations/components/SelectUsers'
 
 // Types
-import {UsersMap} from 'src/organizations/components/Members'
+import {User} from '@influxdata/influx'
+import {UsersMap} from './Members'
```

Comment:
I think the abs. path is better in this case? 'src/organizations/components/Members'

Reply:
I agree!